tests/Backend/SOX/MergeRequestsTest.php

```diff
@@ -93,6 +93,42 @@ public function testPutInReview(): void
         $this->assertEquals('in_review', $mrData['state']);
     }
 
+    public function testMRWorkflowFromDevelopmentToCancel(): void
+    {
+        $oldBranches = $this->branches->listBranches();
+        $this->assertCount(1, $oldBranches);
+
+        $newBranch = $this->branches->createBranch('aaaa');
+
+        $mrId = $this->developerClient->createMergeRequest([
+            'branchFromId' => $newBranch['id'],
+            'branchIntoId' => $oldBranches[0]['id'],
+            'title' => 'Change everything',
+            'description' => 'Fix typo',
+        ]);
+
+        $reviewerClient = $this->getReviewerStorageApiClient();
+        $this->developerClient->mergeRequestPutToReview($mrId);
+
+        $mrData = $reviewerClient->mergeRequestAddApproval($mrId);
+
+        $this->assertEquals('in_review', $mrData['state']);
+        $this->assertCount(1, $mrData['approvals']);
+
+        $mrData = $reviewerClient->mergeRequestAddApproval($mrId);
```

Comment:
viem ze to je draft, len som si to vsimol... nemalo by to takto fungovat nie ? ze jednim tokenom dam 2x approve. to by jeden clovek mohol schvalit cely MR. imho to musia byt dva rozne tokeny

Reply:
jo, mas pravdu, uz to je v connection opravene a tady ten test zacne failovat

Reply:
ha, jeste, ze jsi to pripomnel, protoze jsem mel v connection bug